microsoft-365/enterprise/ms-cloud-germany-transition-phases.md

```diff
@@ -263,18 +264,26 @@ Customers with Dynamics 365 require additional engagement to migrate the organiz
 
 ## Phase 9: Office Apps
 
-**Applies to:** All customers using Office desktop applications (Word, Excel, PowerPoint, Outlook, ...)
+**Applies to:** All customers using Office desktop applications (Word, Excel, PowerPoint, Outlook, OneDrive ...)
+
+In this phase, all client applications and Office Online are performing the client cutover. Azure AD finalizes the tenant scope to point to the Office 365 services and the related endpoints.
 
 Office 365 tenants transitioning to the region "Germany" require all users to close, sign out from Office 365 and back in for all Office desktop applications (Word, Excel, PowerPoint, Outlook, etc.) and OneDrive for Business client after the tenant migration has reached phase 9. Signing out and in, allows the Office services to obtain new authentication tokens from the global Azure AD service.
 
+In case the Office desktop applications will not work after performing signing out and signing in from the applications, it is strongly recommended to run the [Office Client Cutover Tool (OCCT)](https://github.com/microsoft/OCCT) on the affected machine to fix the problem.
```

Comment:
```suggestion
In case the Office desktop applications will not work after performing signing out and signing in from the applications, we strongly recommend running the [Office Client Cutover Tool (OCCT)](https://github.com/microsoft/OCCT) on the affected machine to fix the problem.
```